addons/sale/models/sale_order_line.py

```diff
@@ -1021,6 +1060,19 @@ def _compute_untaxed_amount_to_invoice(self):
 
             line.untaxed_amount_to_invoice = amount_to_invoice
 
+    @api.depends('discount', 'price_total', 'product_uom_qty', 'qty_delivered', 'qty_invoiced_posted')
+    def _compute_amount_to_invoice(self):
+        to_compute = self.company_id.account_use_credit_limit
+        for line in self:
```

Comment:
`self.company_id` can be multiple companies
```suggestion
        to_compute = self.filtered('company_id.account_use_credit_limit')
        (self - to_compute).amount_to_invoice = 0
        for line in to_compute:
```

Reply:
I think it makes more sense to look at the selected company, instead of the company of the SO.
Since all these fields are meant to be used in the context of the credit limit, it makes sense to only do the computations when the setting is actually activated on the selected company, no?
Plus `credit_to_invoice` can only rely on the selected company since there is no company associated with partners, so it seemed logical to use the same condition everywhere :blush: